.github/workflows/uberjar.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       matrix:
         edition: [ee, oss]
-        java-version: [8, 11, 13]
```

Comment:
Now that Java 16 is out, [15 is actually EOL'ed now](https://en.wikipedia.org/wiki/Java_version_history), so it might make sense to change this to 16. 13 is EOL'ed too. 8 and 11 are LTS, and besides that only the current latest version is supported (16). When 17 comes out later this year it will be LTS too.

Reply:
Good point, thanks! I've changed it per your suggestion.